### PR TITLE
Clean migration collections

### PR DESCRIPTION
**Pull request by howbazaar**
Update the comments and sections for pending migration collections.

Also updates the unhappy gofmt.

### CODE REVIEW

File: state/migration_internal_test.go [hunk 1]
```diff
@@ -157,27 +157,40 @@ func (s *MigrationSuite) TestKnownCollections(c *gc.C) {
...
+		// and will be streamed across after migration in a similar way to
+		// logging.
+		auditingC,
+
+		// There is a precheck to ensure that there are no pending reboots
+		// for the model being migrated, and as such, there is no need to
+		// migrate that information.
+		rebootC,
+
+		// Charms as added into the migrated model during the binary transfer
```
Comment (babbageclunk) [inline state/migration_internal_test.go]: typo: as should be are

Comment (howbazaar) [inline state/migration_internal_test.go]: fixed

**Comment by howbazaar** [comment]
$$merge$$

**Comment by howbazaar** [comment]
Another intermittent windows failure $$merge$$

**Pull request merged**